test/Microsoft.Health.Dicom.Web.Tests.E2E/Rest/Audit/AuditTests.cs

```diff
@@ -197,7 +197,7 @@ public async Task GivenStoreRequest_WhenStoringUsingStudyInstanceUid_ThenAuditLo
 
         private async Task ExecuteAndValidate<T>(Func<Task<T>> action, string expectedAction, string expectedPathSegment, HttpStatusCode expectedStatusCode)
         {
-            if (!_fixture.IsUsingInProcTestServer)
+            if (!_fixture.IsInProcess)
             {
                 // This test only works with the in-proc server with customized middleware pipeline
```

Comment:
Do we need to update this comment?

Reply:
Actually never mind, probably not haha

Reply:
Nah, we should be good. I just thought that the name of the property was a bit verbose before